tests/test_examples.py

```diff
@@ -45,6 +45,15 @@ def check_version(rule, target):
                     break
     return True
 
+def check_libraries(rule):
+    for rule_item in rule:
+        try:
+            import rule_item
```

Comment:
```suggestion
            __import__(rule_item)
```
See [here](https://stackoverflow.com/questions/8718885/import-module-from-string-variable) info.

Reply:
thank you, fixed it